src/sentry/runner/initializer.py

```diff
@@ -202,6 +202,17 @@ def bootstrap_options(settings: Any, config: str | None = None) -> None:
     # these will be validated later after bootstrapping
     for k, v in options.items():
         settings.SENTRY_OPTIONS[k] = v
+        # If SENTRY_URL_PREFIX is used in config, show deprecation warning and
+        # set the newer SENTRY_OPTIONS['system.url-prefix']. Needs to be here
+        # to check from the config file directly before the django setup is done.
+        # TODO: delete when SENTRY_URL_PREFIX is removed
+        if k == "SENTRY_URL_PREFIX":
+            warnings.warn(
+                DeprecatedSettingWarning(
+                    "SENTRY_URL_PREFIX", "SENTRY_OPTIONS['%s']" % "system.url-prefix"
```

Comment:
Kinda silly to have the string substitution here since it is not variable. How about just one string? :)